net-utils/src/bin/ip_address_server.rs

```diff
@@ -19,7 +19,7 @@ fn main() {
         .unwrap_or_else(|_| panic!("Unable to parse {}", port));
     let bind_addr = SocketAddr::from(([0, 0, 0, 0], port));
     let tcp_listener = TcpListener::bind(bind_addr).expect("unable to start tcp listener");
-    let _runtime = solana_net_utils::ip_echo_server(tcp_listener);
+    let _runtime = solana_net_utils::ip_echo_server(tcp_listener, /*shred_version=*/ None);
```

Comment:
I think it'd be ok if hard coded a shred_version here, say "0" or "123", which I think would allow the removal of the `Option` from `ip_echo_server()`